src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -103,5 +105,47 @@ public static byte[] GenerateKeyBytes(int sizeInBits)
 
             return key;
         }
+
+        /// <summary>
+        /// Gets the DateTime using the number of seconds from 1970-01-01T0:0:0Z (UTC)
+        /// </summary>
+        /// <param name="key">Claim in the payload that should map to an integer, float, or string.</param>
+        /// <param name="payload">The payload that contains the desired claim value.</param>
+        /// <remarks>If the claim is not found, the function returns: DateTime.MinValue
+        /// </remarks>
+        /// <exception cref="FormatException">If the value of the claim cannot be parsed into a long.</exception>
+        /// <returns>The DateTime representation of a claim.</returns>
+        public static DateTime GetDateTime(string key, JObject payload)
+        {
+            JToken jToken;
+            if (!payload.TryGetValue(key, out jToken))
+                return DateTime.MinValue;
+
+            long dateValue = ParseTimeValue(jToken, key);
```

Comment:
could lose the dateValue variable and inline it into 
....Convert.ToDouble(ParseTimeValue(jToken, key),

Reply:
Done.